doc/nrf/conf.py

```diff
@@ -85,6 +85,7 @@
               'interbreathe',
               'table_from_rows',
               'options_from_kconfig',
+              'ncs_include',
```

Comment:
could `ncs_include` be a too generic name, that we could want to use for other python packages in future ?

Reply:
but this doesn't collide with python packages, because the python package, if uploaded to pip is always prefixed with`sphinx-` or `sphinxcontrib-`

Reply:
What you see here is the name of the extension, not the package. @utzig can confirm of course.

Reply:
Correct, it's the extension's name, I suppose it could be generalized and useful for other projects, but I had no intention of publishing it as a package since it was tailored for NCS, and I am not sure if anyone else uses a similar setup.

Reply:
I know this is not a package.
I'm not thinking on publishing this one, more in case there is a second `ncs_include` package in future.

I don't know exactly the code for sphinx, but I assume it imports all extensions from this `extensions` variable.
And I do see that we have:
```
sys.path.append(os.path.join(NRF_BASE, 'scripts', 'sphinx_extensions'))
```
so in case there was ever to be second `ncs_include` package installed in the system, I believe there could be a risk of collision.

So I just like to play it safe :wink:

Reply:
I think it would be better to create a "nrf" (or "ncs") namespace, like Zephyr also does for its extensions, and include this one, "boards_from_rows", "kconfig_from_options", "interbreathe" (which I am not sure is still required with latest Sphinx), and possibly others. But I sure don't want to move things around here! :-)